src/components/MarkdownProvider/index.tsx

```diff
@@ -48,12 +62,26 @@ export const MarkdownProvider: React.FC = ({ children }) => (
         CodeExample,
         PackageDescription,
         PropSheets,
-        h2: H2,
-        h3: H3,
-        h4: H4,
-        h5: H5,
-        h6: H6,
-        p: P
+        h2: StyledH2,
+        h3: StyledH3,
+        h4: StyledH4,
+        h5: StyledH5,
+        h6: StyledH6,
+        p: StyledP,
+        blockquote: StyledBlockquote,
+        hr: StyledHr,
+        inlineCode: Code,
+        pre: StyledPre,
+        code: MDSyntaxHighlighter,
+        ul: UL,
+        ol: OL,
+        li: LI,
+        table: Table,
+        thead: THead,
+        tbody: TBody,
+        tr: TR,
+        td: TD,
+        th: TH
```

Comment:
Please update to map `strong` to a styled component that renders `<b>` – ideally using the new typography [isBold](https://garden.zendesk.com/react-components/typography/#md) styling.

Also, update to map `em` to a styled component that renders `<i>`. Garden opinion is that `<b>` and `<i>` are the correct elements for stylistic rendering. Since we can't infer semantics from markdown, we'll pass on  the `<strong>` and `<em>` tags.

The most recent v8.0.0 version of `css-bedrock` bears this out.

Reply:
After discussing yesterday with design, I'm not sure this is correct for our specific usages.

Within a component boundary, usage of `<strong>` and `<em>` would be invalid as the visualization is only stylistic. But, our markdown provider, and for any long-form text within a product, the `<strong>` and `<em>` tags are required to denote differences in text for a screen reader.

Additionally, markdown doesn't provide styling, it only provides semantics. The markdown schema denotes these as [emphasis](https://github.com/syntax-tree/mdast#emphasis) and [strong](https://github.com/syntax-tree/mdast#strong). I think we should enforce this here.

Reply:
I can buy this. Thanks for the explanation.